tests/lib/apt.sh

```diff
@@ -11,6 +11,9 @@ install_build_snapd(){
         apt install -y --only-upgrade snapd
         mv sources.list.back /etc/apt/sources.list
         apt update
+        if [ "$SPREAD_REBOOT" = 0 ]; then
```

Comment:
Suggestion: perhaps this could be written like:
[ "$SPREAD_REBOOT" = 0 ] && REBOOT